### PR TITLE
Features/animations performance

### PR DESCRIPTION
**Pull request by rodrigo-cervone**
Making some tests around UI performance.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -42,4 +49,7 @@ export function provideStorage() {
   ]
 })
 export class AppModule {
+  constructor() {
+    // console.log(UserLoginPage)
```
Comment (jhcruz) [inline src/app/app.module.ts]: commented code...

**Review by jhcruz — APPROVED** [review]
Almost approved, only check a commented code and conflicts...

File: src/app/app.module.ts (hunk 1 above)
Comment (rodrigo-cervone) [reply src/app/app.module.ts]: good catch

**Comment by urieljbm** [comment]
All looks fine.

**Pull request merged**